ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -746,4 +746,6 @@ literals:
 - zone_air_voc_percentage_setpoint
 - zone_air_voc_percentage_sensor
 
-
+# Added for SG-SIN-MBC2
```

Comment:
Please remove site-specific reference.

Reply:
@JennyChan-ptpl can you please apply the changes?